### PR TITLE
Flesh out finale

### LINKED ISSUE #522: Expanding the finale to revisit profiles
From the review by @ChristinaLK:

> An "if there's time/energy" suggestion: in the final chapter,
 revisit the profiles from the first chapter, and what chapters of
the books were helpful to each person in achieving their goals. E.g.
"Amira now has her scripts organized in a repository, following the
project scheme described in this book. She made a documentation page
for her repository so her co-workers are able to see the work she's
done..." etc. Not necessary, but would be fun. :)

I'm going to try to flesh out our one sentence wrap up.

### PR DESCRIPTION
**Pull request by cwickham**
Resolves #522. Harder to write than I thought....

### CODE REVIEW

**Comment by DamienIrving** [comment]
Once you've been exposed to all this stuff it can take a long time to work it all into your daily workflow (e.g. after attending my first Software Carpentry workshop it was 8 months before I finally started using version control). To acknowledge this reality, maybe one of the characters should have a new concept from the book that is on their to-do list but they haven't done it yet?

File: chapters/finale.Rmd [hunk 1]
```diff
@@ -5,6 +5,48 @@
...
+in version control, 
+and on GitHub. 
+Her work has already paid off:
+a colleague spotted a problem in an analysis step,
+that impacted five figures in one of her reports.
+Because Amira has embraced GitHub and Make,
+the colleague could easily suggest a fix through a Pull Request,
+and Amira could easily regenerate all of the affected figures.
+
+She used to be intimidated by the unix shell,
```
Comment (gvwilson) [inline chapters/finale.Rmd]: "Unix"

File: chapters/finale.Rmd [hunk 2]
```diff
@@ -5,6 +5,48 @@
...
 We have come a long way since we first met Amira, Jun, and Sami in Section \@ref(intro-personas).
+
+Amira now has scripts, data sets and reports organized, 
+in version control, 
+and on GitHub. 
+Her work has already paid off:
+a colleague spotted a problem in an analysis step,
+that impacted five figures in one of her reports.
+Because Amira has embraced GitHub and Make,
+the colleague could easily suggest a fix through a Pull Request,
```
Comment (gvwilson) [inline chapters/finale.Rmd]: "pull request" (lower case)

File: chapters/finale.Rmd [hunk 3]
```diff
@@ -5,6 +5,48 @@
...
+and issues countless commands to Git and Make. 
+Her comfort with the shell is also helping 
+as she learns how to run her analyses on a remote computing cluster
+from Sami.
+
+Sami had experience with Git and Make in software projects from
+their undergraduate studies, 
+but they have a new appreciation for their importance in research projects. 
+They've reworked their Git and Make workshops to use examples of data pipelines, 
+and have been getting rave reviews from participants!
```
Comment (gvwilson) [inline chapters/finale.Rmd]: I would use "." instead of "!", but we Canadians are an understated people...

File: chapters/finale.Rmd [hunk 4]
```diff
@@ -5,6 +5,48 @@
...
+
+Sami has also gained an appreciation 
+for the importance of provenance 
+of both data and code in research. 
+They've been helping their users 
+by suggesting ways they can make their research more accessible, inspectable and reproducible. 
+
+Jun is now working on his first python package.
```
Comment (gvwilson) [inline chapters/finale.Rmd]: "Python"

File: chapters/finale.Rmd [hunk 5]
```diff
@@ -5,6 +5,48 @@
...
+They've been helping their users 
+by suggesting ways they can make their research more accessible, inspectable and reproducible. 
+
+Jun is now working on his first python package. 
+He's added better error handling so he and his
+users get better information when something goes wrong,
+he's implemented some testing strategies to give him and his users confidence his code works, and
+he's improved his documentation.
+
+Jun has also added a license, a code of conduct and contributing guidelines to his project repo.
```
Comment (gvwilson) [inline chapters/finale.Rmd]: We have title-cased "Code of Conduct" elsewhere in the book.

File: chapters/finale.Rmd [hunk 6]
```diff
@@ -5,6 +5,48 @@
...
+
+Jun is now working on his first python package. 
+He's added better error handling so he and his
+users get better information when something goes wrong,
+he's implemented some testing strategies to give him and his users confidence his code works, and
+he's improved his documentation.
+
+Jun has also added a license, a code of conduct and contributing guidelines to his project repo. 
+He's already had a contribution that fixes some typos
+in the documentation!
```
Comment (gvwilson) [inline chapters/finale.Rmd]: Again, we've mostly avoided '!' in the book - quiet café jazz rather than the flaming guitars of our youth, kind of thing.

**Review by gvwilson — APPROVED** [review]
Very minor typographical suggestions, then please go ahead and merge - thank you.

**Comment by cwickham** [comment]
@DamienIrving Great point!  I switched Make from Amira's done to to-do list.

**Comment by cwickham** [comment]
Ready for merge

**Pull request merged**